### PR TITLE
python3Packages.pycomfoconnect: init at 0.4

### PR DESCRIPTION
**Pull request by fabaff**
###### Motivation for this change
Python module to interact with ComfoAir Q350/450/600 units

https://github.com/michaelarnauts/comfoconnect

This is Home Assistant dependency.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 119126` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>3 packages built:</summary>
  <ul>
    <li>home-assistant</li>
    <li>python38Packages.pycomfoconnect</li>
    <li>python39Packages.pycomfoconnect</li>
  </ul>
</details>

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 119126` at e9a0be70 run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>3 packages built successfully:</summary>
  <ul>
    <li>home-assistant</li>
    <li>python38Packages.pycomfoconnect</li>
    <li>python39Packages.pycomfoconnect</li>
  </ul>
</details>
<details>
  <summary>1 suggestion:</summary>
  <ul>
    <li>warning: unused-argument

Unused argument: `pytestCheckHook`.
Near pkgs/development/python-modules/pycomfoconnect/default.nix:5:3:
```
  |
5 | , pytestCheckHook
  |   ^
```
</li>
  </ul>
</details>

File: pkgs/development/python-modules/pycomfoconnect/default.nix [hunk 1]
```diff
@@ -0,0 +1,34 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, protobuf
+, pytestCheckHook
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/pycomfoconnect/default.nix:5]:
```suggestion
```

**Comment by Mic92** [comment]
Result of `nixpkgs-review pr 119126` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package failed to build:</summary>
  <ul>
    <li>home-assistant</li>
  </ul>
</details>
<details>
  <summary>2 packages built:</summary>
  <ul>
    <li>python38Packages.pycomfoconnect</li>
    <li>python39Packages.pycomfoconnect</li>
  </ul>
</details>

**Comment by Mic92** [comment]
/nix/store/3a9vs2wy4mqfjy0svvy3m8ddmpryv5hc-python3.8-pywemo-0.6.4.drv failed for home-assistant so it might be unrelated.

**Comment by mweinelt** [comment]
Hm, ofBorg also transiently was unhappy about pywemo in https://github.com/NixOS/nixpkgs/pull/116256, but I could never reproduced that, neither on x86_64 nor on aarch64. Can you get a log? @Mic92

**Comment by Mic92** [comment]
@mweinelt https://p.krebsco.de/18z0khc

**Comment by mweinelt** [comment]
Right, somewhat racy sqlite queries. Let's disable that test and move on.

File: pkgs/development/python-modules/pycomfoconnect/default.nix (hunk 1 above)
Comment (mweinelt) [reply pkgs/development/python-modules/pycomfoconnect/default.nix:5]: @fabaff?

**Comment by mweinelt** [comment]
https://github.com/NixOS/nixpkgs/pull/119126/files#r611199763

Unaddressed review comment right here.

**Pull request merged**